src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsEngine.java

```diff
@@ -25,15 +25,24 @@
 import java.util.*;
 import java.util.stream.Collectors;
 
+/**
+ * the core engine for the GenotypeGVCF that does all of the actual work of the tool
```

Comment:
`The core engine for the GenotypeGVCF that does all of the actual work of the tool` -> 'Engine class to allow for other classes to replicate the behavior of GenotypeGVCFs. See {@link GenotypeGVCFs} for details.